.gitignore

```diff
@@ -6,7 +6,6 @@ __pycache__/
 .pytest_cache/
 .pyre/
 .vagrant/
-.watchmanconfig
```

Comment:
I'll remove this - but I'm making a note to myself we need to add a note to create one of these files in the `Repo Setup` or something